dash/dash.py

```diff
@@ -805,6 +811,23 @@ def wrap_func(func):
             def add_context(*args, **kwargs):
 
                 output_value = func(*args, **kwargs)
+                setattr(
```

Comment:
For each callback, we can actually update the component object with the correct props so that the layout in the backend matches the layout in the front-end. This is not really necessary for this PR, but may be useful in the future.